code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -10,3 +10,44 @@
 	liked_food = JUNKFOOD | FRIED
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	payday_modifier = 1
+
+	var/groin_pain_timer
+
+	var/groin_pain_threshold = 20
+	var/groin_pain_nads_modifier = 2
+
+/datum/species/human/handle_groin_damage(mob/living/carbon/human/H, damage_amount, sharpness)
+	if(damage_amount >= 10)
+		H.emote("wince")
+
+	if (sharpness != SHARP_NONE)
+		return
+
+	var/pain_threshold = groin_pain_threshold
+	if (H.gender == MALE)
+		pain_threshold -= groin_pain_nads_modifier
```

Comment:
wait a minute oh god there's no way for this to regenerate

if you punch someone in the dick enough, you can make it so that any damage to their groin will cause them to get staminacritted in 8 seconds, and they can't remove this by any means except a body swap or a race change (or maybe some monkey shenanigans?).

Reply:
pain_threshold resets everytime handle_groin_damage is called

Reply:
ah